### PR TITLE
add max-width to all content

### PR DESCRIPTION
**Pull request by elbotho**
@Entkenntnis this is bit crude, but seems effective. Any objections? :)

### CODE REVIEW

**Comment by Entkenntnis** [comment]
unfortunately, this breaks the sticky placement of the share-toolbox.

**Comment by Entkenntnis** [comment]
maybe we can find another way to make the sticky placement possible & hide overflow

**Comment by elbotho** [comment]
> unfortunately, this breaks the sticky placement of the share-toolbox.

😿 
nice catch.

File: src/components/header-footer.tsx [hunk 1]
```diff
@@ -11,7 +11,9 @@ export function HeaderFooter({ children }: HeaderFooterProps) {
   return (
     <>
       <Header />
-      <div className="min-h-[68vh] max-w-full overflow-x-hidden">
+      <div
+        className="min-h-[68vh] max-w-full overflow-x-hidden lg:overflow-visible" /* needed for sticky menu*/
```
Comment (Entkenntnis) [inline src/components/header-footer.tsx:15]: @elbotho THIS could be the final approach :)

Comment (Entkenntnis) [reply src/components/header-footer.tsx:15]: only thing that breaks is the sticky header for e.g. https://de.serlo.org/entity/repository/compare/224015/224311 - but that would be fine.

Comment (elbotho) [reply src/components/header-footer.tsx:15]: since it's working on larger screens I can live with that.

**Pull request merged**